content/applications/finance/payment_providers/paypal.rst

```diff
@@ -2,120 +2,78 @@
-preferences --> Update`, scroll down to :guilabel:`Payment data transfer` and select :guilabel:`On`.
-
-.. tip::
-   PayPal displays your **PDT Identity Token** as soon as :ref:`Auto return <paypal/auto-return>`
-   and :ref:`Payment Data Transfer (PDT) <paypal/pdt>` are enabled. If you need the **PDT Identity
-   Token**, disable and re-enable :guilabel:`Payment data transfer` to display the token again.
-
-PayPal Account Optional
------------------------
-
-We advise not to prompt customers to log in with a PayPal account upon payment. It is better and
-more accessible for customers to pay with a debit/credit card. To disable that prompt, go to
-:menuselection:`Account Settings --> Website payments --> Update` and select :guilabel:`On` for
-:guilabel:`PayPal account optional`.
-
-Payment Messages Format
------------------------
-
-If you use accented characters (or anything other than primary Latin characters) for customer names
-or addresses, then you **must** configure the encoding format of the payment request sent by Odoo to
-PayPal. If you do not, some transactions fail without notice.
-
-To do so, go to `your production account <https://www.paypal.com/cgi-bin/customerprofileweb
-?cmd=_profile-language-encoding>`_. Then, click :guilabel:`More Options` and set the two default
-encoding formats as :guilabel:`UTF-8`.
+   If you are trying PayPal as a test, access your :ref:`PayPal Sandbox account
+   <payment_providers/paypal/testing>` and `configure the encoding format for your sandbox account
+   <https://sandbox.paypal.com/cgi-bin/customerprofileweb?cmd=_profile-language-encoding>`_.
 
 .. tip::
-   - For Encrypted Website Payments & EWP_SETTINGS error, please check the `Paypal documentation
-     <https://developer.paypal.com/docs/online/>`_.
-   - Configure your :ref:`Paypal Sandbox account <paypal/testing>`, then follow this
-     `link <https://sandbox.paypal.com/cgi-bin/customerprofileweb?cmd=_profile-language-encoding>`_
-     to configure the encoding format in a test environment.
-
-Settings in Odoo
-================
-
-.. seealso::
-   :ref:`payment_providers/add_new`
+   For encrypted website payments & EWP_SETTINGS errors, please check the `PayPal documentation
+   <https://developer.paypal.com/docs/online/>`_.
 
-Odoo needs your **API Credentials** to connect with your PayPal account. To do so, go to
-:menuselection:`Accounting --> Configuration --> Payment Providers` and :guilabel:`Activate` PayPal.
-Then, enter your PayPal account credentials in the :guilabel:`Credentials` tab:
+.. _payment_providers/paypal/configuration-odoo:
 
-- :guilabel:`Email`: the login email address in Paypal;
-- :guilabel:`PDT Identity Token`: the key used to verify the authenticity of transactions.
+Configuration in Odoo
+=====================
 
-.. _paypal/testing:
+#. :ref:`Navigate to the payment provider PayPal <payment_providers/add_new>`,
+   change its state to :guilabel:`Enabled`, and make sure it is :guilabel:`Published`.
```

Comment:
I'd move that in last step because enabling a provider makes its credentials fields required (red) and prevents saving the form or clicking a button until the fields are filled in.